### ISSUE TITLE
Data Grid Help Wanted

### ISSUE THREAD

**Issue by Samirgc** [enisn/UraniumUI#70]
Uranium UI Data Grid 
Is there a way to change the title color of data grid ? 

![Noway](https://user-images.githubusercontent.com/35827732/195339459-6f746d0a-1e19-44ac-9d12-2be77af33b8d.png)

**Comment by enisn** [comment]
Oh, it seems there is no customization way for it. I'll create a customization logic for Title cells.

But now, you can set the entire labels via setting resource dictionary.

```xml
<material:DataGrid ItemsSource="{Binding Items}">
    <material:DataGrid.Resources>
        <Style TargetType="Label">
            <Setter Property="TextColor" Value="Red" />
        </Style>
    </material:DataGrid.Resources>
</material:DataGrid>
```

It'll affect to all labels of datagrid, but you can define a custom item template for cells and customize cell labels.

I'll add a header item template for datagrid.

**Comment by Samirgc** [comment]
Thanks this will server my need!  But improvement is appreciated. Again Thanks ! 
![Thanks](https://user-images.githubusercontent.com/35827732/195387493-fe1c0e2d-7a9c-452c-84d0-95fbc7af96bf.PNG)

**Comment by enisn** [comment]
Closed due to #71
